packages/schematics/angular/utility/workspace-models.ts

```diff
@@ -26,11 +26,14 @@ export enum Builders {
   Prerender = '@angular-devkit/build-angular:prerender',
   BrowserEsbuild = '@angular-devkit/build-angular:browser-esbuild',
   Karma = '@angular-devkit/build-angular:karma',
+  BuildKarma = '@angular/build:karma',
   TsLint = '@angular-devkit/build-angular:tslint',
   NgPackagr = '@angular-devkit/build-angular:ng-packagr',
   BuildNgPackagr = '@angular/build:ng-packagr',
   DevServer = '@angular-devkit/build-angular:dev-server',
+  BuildDevServer = '@angular/build:dev-server',
   ExtractI18n = '@angular-devkit/build-angular:extract-i18n',
+  BuildExtractI18n = '@angular/build:extract-i18n',
   Protractor = '@angular-devkit/build-angular:private-protractor',
   BuildApplication = '@angular/build:application',
```

Comment:
Consider: I see that `X` and `BuildX` is an existing convention, but I wonder if it would be worth renaming the existing ones to `DevkitX`, `XDevkit`, or something similar to be more clear about the distinction between them.

Alternatively we could potentially drop the `BuildX` prefix with that being the "default" option we want to use, with `DevkitX` being the legacy versions.

If we do make this change, it's probably worth a separate PR, but something to think about.

Reply:
While these are "internal", they have been and probably still are used by third-party custom schematics.  This makes more comprehensive changes problematic.  I do like the `Devkit` variation though.